Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/ui/main/MainActivity.java

```diff
@@ -248,25 +237,6 @@ protected void onActivityResult(int requestCode, int resultCode, Intent result)
     }
   }
 
-  @Override
-  public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions,
-          @NonNull int[] grantResults)
-  {
-    super.onRequestPermissionsResult(requestCode, permissions, grantResults);
-
-    if (requestCode == PermissionsHandler.REQUEST_CODE_WRITE_PERMISSION)
-    {
-      if (grantResults[0] == PackageManager.PERMISSION_DENIED)
-      {
-        PermissionsHandler.setWritePermissionDenied();
```

Comment:
This is no longer being called anywhere. Are you sure this is proper?

Reply:
Oh ok, I see. I'll set that in the SplashScreen activity.